database/postgres/pg.test.js

```diff
@@ -0,0 +1,80 @@
+jest.mock('pg')
+const {Client} = require('pg')
+const {startPGDB, closePGDB, createPgAccount, deletePgAccount} = require('./pg')
+
+describe('Test PG DB', ()=>{
+  afterEach(()=>{
+    jest.clearAllMocks()
+  })
+  let mockClient = {
+    query: jest.fn().mockReturnValue(Promise.resolve()),
+    connect: jest.fn().mockReturnValue(Promise.resolve()),
+    end: jest.fn().mockReturnValue(Promise.resolve())
+  }
+  Client.mockImplementation(function(){
+    return mockClient
+  }) 
+  global.console.log = jest.fn() 
```

Comment:
I think it's `console.log = jest.fn()`, also it'll be clearer if it moves to under 47th line

Reply:
but that wouldn't make sense for line 75 where it's called again. Since it's global i'm changing the functionality of `console` so I wanted to make it clear that this is changing the functionality of console for the whole database file. You do you have a good point so I'm going to take it outside of the postgres test so we can use it in the MongoDB and Neo4J test as well.

Reply:
this should be scoped into the function itself. Its not good for a developer later on to try to `console.log` something, but nothing shows up. This will essentially remove `console.log` functionality, which will remove debugging feature for alot of devs. Better to be inside the test.